src/amplitude-wrapper.js

```diff
@@ -12,6 +12,23 @@
           var r = t.createElement("script");
           r.type = "text/javascript", r.integrity = "sha384-QhZkEQJe2NFJ4yDkn/RFnD+NP0FINrep4tUh958v8McXRqszeRUQWbwBCfFqZvnF", r.crossOrigin = "anonymous", r.async = !0, r.src = "https://cdn.amplitude.com/libs/marketing-analytics-browser-0.2.0-min.js.gz", r.onload = function() {
               e.amplitude.runQueuedFunctions || console.log("[Amplitude] Error: could not load SDK")
+              var gtmLibraryPlugin = () => {
+                  return {
+                      name: 'gtm-library-enrichment',
+                      type: 'enrichment',
+                      setup: async () => undefined,
+                      execute: async (event) => {
+                          event['library'] = `amplitude-ts-gtm/3.0.0-beta.4`
```

Comment:
can we import the version from `package.json`? i think if you check main branch, there's a sample of how this file imports the version from `package.json`

Reply:
Thanks! Good tip, updated in the [654b7a6](https://github.com/amplitude/amplitude-js-gtm/pull/9/commits/654b7a6ec0dcbde070bf9129c748eb1a70219945).